onebusaway-android/src/androidTest/java/org/onebusaway/android/nav/test/NavigationTest.java

```diff
@@ -101,23 +101,21 @@ public void testTrip2C() throws IOException {
      * Destination Stop: Dale Mabry Hwy @ Hudson Ln @ Taco Bell
      * Recorded In: Bus (Route 36) 4 stops
      * Device Used: Nexus 5
-     * FIXME - Currently fails with "Pull the cord triggered too soon"
      */
     @Test
     public void testTrip3() throws IOException {
-        runSimulation("nav_trip3", 95, 115);
+        runSimulation("nav_trip3", 95, 109);
     }
 
     /**
      * Started Stop: Dale Mabry Hwy @ Linebaugh Av
      * Destination Stop: Dale Mabry Hwy @ Hudson Ln @ Taco Bell
      * Recorded In: car following Bus (Route 36) 4 stops
      * Device Used: Nexus 5
-     * FIXME - Currently fails with "Pull the cord triggered too soon"
      */
     @Test
     public void testTrip3C() throws IOException {
-        runSimulation("nav_trip3c", 87, 109);
+        runSimulation("nav_trip3c", 87, 106);
```

Comment:
It looks like 109 or 110 are the GPS IDs that should have triggered the "pull the cord now" alert, if the algorithm worked perfectly.  Again, here ID 106 is the point that actually triggered the "pull the cord now" alert with the current algorithm, but this was too early (i.e., the user would have gotten off at the stop prior to the destination stop, not the destination stop).

![image](https://user-images.githubusercontent.com/928045/45649333-01eef400-ba99-11e8-879e-ccab8c781552.png)